server/join_test.go

```diff
@@ -53,24 +53,33 @@ func newTestMultiJoinConfig(count int) []*Config {
 
 func waitMembers(svr *Server, c int) error {
 	// maxRetryTime * waitInterval = 5s
-	maxRetryCount := 500
-	waitInterval := 100 * time.Millisecond
+	maxRetryCount := 10
+	waitInterval := 500 * time.Millisecond
+	ctx, cancel := context.WithTimeout(context.Background(), defaultDialTimeout)
+	defer cancel()
+
+	client := svr.GetClient()
+	for ; maxRetryCount != 0; maxRetryCount-- {
```

Comment:
I am not quite sure what you mean, `c` is how many members we have to wait, maybe I should remove `count` in [line 68](https://github.com/pingcap/pd/pull/239/files#diff-76765a78a72ec5f47ca5e941d971deb2R68)